templates/data/opensearch/index.html

```diff
@@ -0,0 +1,530 @@
+        <div class="p-card--overlay">
+          <div class="row--50-50">
+            <div class="col">
+              <h3 class="u-align-text--left u-no-margin--bottom">Managed</h3>
+            </div>
+            <div class="col">
+              <h3 class="u-align-text--right u-no-margin--bottom">From $3099</h3>
+            </div>
+          </div>
+          <p class="u-align-text--right">Per node, per year</p>
+          <p>
+            Get there faster. Let us deploy and operate your OpenSearch cluster, in your cloud tenancy or in your data centre.
+          </p>
+          <hr class="p-rule is-muted" />
+          <div class="p-section--shallow  u-no-margin--bottom">
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">Backed by an SLA</li>
+              <li class="p-list__item is-ticked">MSP and SOC2 certified service</li>
+              <li class="p-list__item is-ticked">Low cost of ownership</li>
+            </ul>
+            <p>
+              <a href="/data/opensearch/managed" aria-controls="data-spark-modal">Explore Managed OpenSearch Solutions&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="col-6 col-medium-3">
+        <hr class="p-rule--highlight" />
+        <div class="p-card--overlay p-section">
+          <div class="row--50-50">
+            <div class="col">
+              <h3 class="u-align-text--left u-no-margin--bottom">Self-managed</h3>
+            </div>
+            <div class="col">
+              <h3 class="u-align-text--right u-no-margin--bottom">From $500</h3>
+            </div>
+          </div>
+          <p class="u-align-text--right">Per node, per year</p>
+          <p>
+            24/7, weekday or advanced fire-fighting support for OpenSearch from Canonical for up to 10 years per release track.
+          </p>
+          <hr class="p-rule is-muted" />
+          <div class="p-section--shallow  u-no-margin--bottom">
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">Expert troubleshooting on hand</li>
+              <li class="p-list__item is-ticked">Security and bug fixes</li>
+              <li class="p-list__item is-ticked">Full stack coverage</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>OpenSearch use cases</h2>
+      </div>
+      <div class="col">
+        <p>
+          OpenSearch is an open-source search and analytics suite. Developers build solutions for search, data observability, data ingestion and more.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/data/opensearch/what-is-opensearch">Learn more about OpenSearch&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+    <div class="row--25-75 is-split-on-medium">
```

Comment:
This class has been deprecated, it's responsive by default now
```suggestion
    <div class="row--25-75">
```